src/app/views/infoblock/infoblock.html

```diff
@@ -0,0 +1,4 @@
+<div class="panel-content text-center">
+  <h2 class="{{ $ctrl.colorClass }}">{{ $ctrl.value }}</h2>
```

Comment:
Any time you're adding a class via angular, you should use the `ng-class` directive, rather than templating the values into the class attribute.